### PR TITLE
Run3-hcx355 Trying out the most recent zdc description in the setup

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Trying out the most recent description of the ZDC detector in the setup

#### PR validation:

Use specific workflows to test this using the ones in SimG4CMS/Calo/test/python

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

File: SimG4CMS/Calo/plugins/ZDCSimHitStudy.cc [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  indx_ = tfile->make<TH1F>("Indx", "Hit ID", 100, 0., 100);
+}
+
+void ZDCSimHitStudy::analyze(const edm::Event &e, const edm::EventSetup &) {
+  edm::LogVerbatim("HitStudy") << "ZDCSimHitStudy::Run = " << e.id().run() << " Event = " << e.id().event();
+
+  bool getHits = false;
+  std::vector<PCaloHit> zdcHits;
+  const edm::Handle<edm::PCaloHitContainer> &hitsCalo = e.getHandle(toks_calo_);
+  if (hitsCalo.isValid())
```
Comment (iarspider) [inline SimG4CMS/Calo/plugins/ZDCSimHitStudy.cc:93]: @bsunanda can this (lines 90, 93, 94) be simplified to `bool getHits = histCalo.isValid();`?

Comment (bsunanda) [reply SimG4CMS/Calo/plugins/ZDCSimHitStudy.cc]: Taken care of

**Comment by bsunanda** [comment]
@civanch Please approve this

**Comment by bsunanda** [comment]
@civanch Please approve this PR

**Comment by civanch** [comment]
please test

**Comment by civanch** [comment]
+1

**Pull request merged**